docs/getting_started/ibexa_cloud_guide.md

```diff
@@ -0,0 +1,119 @@
+
+![How does Ibexa Cloud work](ibexa_cloud.png)
+
+## [[= product_name_cloud =]] account
+
+If you want to use [[= product_name_cloud =]], you need to follow [installation process](install_on_ibexa_cloud.md) and check the [requirements](requirements.md#ibexa-cloud-requirements-and-setup).
+
+To use [[= product_name_cloud =]], you must make arrangements with [[= product_name_base =]] to get and set up a user account.
+To do it, contact [[= product_name_base =]] team by filling up the form available at this link: https://www.ibexa.co/about-ibexa/contact-us.
+
+When you have an account, you can log in to https://console.ibexa.cloud.
+
+## Capabilities
+
+### Platform as a Service
+
+[[= product_name_cloud =]] is a PaaS provider. It's a cloud-based subscription service that you can use for developing, managing, and running applications without infrastructure concerns.
+This cloud computing approach gives users access to a full cloud platform, including hardware, software, and infrastructure.
+PaaS eliminates the requirement to buy and install the necessary hardware and software. All you have to do is access it and you can start deploying resources and developing right away.
+
+![How PaaS works](how_paas_works.png)
+
+### Automation
+
+[[= product_name_cloud =]] automates time-consuming testing processes and encourages ongoing QA testing, so your projects are ready for deployment much faster.
+For further protection and simple rollbacks, it also automates your backup procedures.
+This guarantees that the developers are working within a framework that is adaptable, safe, and responsive to their requirements.
+Thanks to built-in CI/CD features, you can also reduce the need for manual testing and accelerate your development.
+
+![Ibexa Cloud - effort](ibexa_cloud_effort.png)
+
+### Digital tool
+
+[[= product_name_cloud =]] is a digital tool - it's a perfect starting step toward your company's digital transformation.
+What's more, [[= product_name_cloud =]] as an end-to-end cloud hosting platform, it's scalable, requires little up-front expenditure, and no ongoing maintenance.
+[[= product_name_cloud =]] as a PaaS solution enables you to start constructing effective digital systems that provide great consumer experiences.
+
+## Benefits
+
+### Safety and simplicity
+
+Comprehensive data security procedures guarantee that you maintain complete ownership of your client's data, defining where it's stored.
+You can be sure that the safety protocols are compliant with all applicable legislation.
+What's more, all updates to your code and infrastructure are fully auditable. Global CDN (Content Delivery Network) is included and fully managed.
+[[= product_name_cloud =]], thanks to grid architecture and extensive [[= product_name_base =]] support, enables effortless deployment.
+You can create a clone not only of the code, but also data, as well as the service, and the infrastructure.
```

Comment:
```suggestion
You can create a clone not only of the code, but also data, and the infrastructure.
```

Nitpick: TBH I'm not sure what "service" means in this context - it's not code, data and infrastracture 🤔